editor.js

```diff
@@ -3,7 +3,7 @@
 	Extension Editor for XKit 7
 	Version 1.0
 
-	(c) 2011 - 2013 STUDIOXENIX
+	(c) 2011 - 2015 STUDIOXENIX & New-XKit Team
```

Comment:
Given that https://github.com/new-xkit/XKit/commit/06b2805f76be5d4948c18811b27ac8d4f9e166fc is Atesh's last commit we have in `master` on October 18th, 2014 and https://github.com/new-xkit/XKit/commit/c4973264a0cfffe0a333c26a9b578ea5078bb753 is the first New-XKit Team commit to the repo on July 7th, 2015, I stand by my original suggestion.